### ISSUE TITLE
[iOS] 'RCTBridgeModule.h' file not found

### ISSUE THREAD

**Issue by urikphytech** [negativetwelve/react-native-heap-analytics#2]
After installing library using cocoa pods i'm getting the following error:

`'RCTBridgeModule.h' file not found` in `RNHeap.h`

Here are some of my configurations:
Podfile:

```
platform :ios, '8.0'

target 'PhytechApp' do
  pod "react-native-heap-analytics", path: "../node_modules/react-native-heap-analytics"
end

```

Header search paths include:
`$(SRCROOT)/../node_modules/react-native-heap-analytics/ios (non-recursive)`

Anything else I can provide?
Thanks!
Uri

**Comment by negativetwelve** [comment]
Hi @urikphytech, thanks for writing in. What version of react-native are you on? This module only works with <0.40 unfortunately. However, let me take a look at what it would take to upgrade. I'll post here with an update soon

**Comment by urikphytech** [comment]
Thanks for the response. I'm using 0.42.0

**Comment by negativetwelve** [comment]
Released version 1.0.0, want to give it a try @urikphytech? https://www.npmjs.com/package/react-native-heap-analytics

**Comment by urikphytech** [comment]
@negativetwelve Thanks! Now i'm getting this error:

```
/Users/uriklar/Code/MyProject/node_modules/react-native-heap-analytics/ios/RNHeap.m:11:9: fatal error: 'Heap.h' file not found
#import "Heap.h"
```

**Comment by negativetwelve** [comment]
yeah that's the main library not being found - you're using cocoapods correct? Make sure you run `pod install` and also check that you set the header search paths to recursive for the `node_modules/react-native-heap-analytics`

**Comment by urikphytech** [comment]
Yeah, initialized a podfile and added the code you mentioned in the readme. 

I changed the header search path to recursive, and now i'm getting the original error again :-/
```'React/RCTBridgeModule.h' file not found``` in ```RNHeap.h```

**Comment by negativetwelve** [comment]
If you expand the `Pods` project, are you able to see Heap here? 

![image](https://user-images.githubusercontent.com/1435378/28646655-b6b5d56e-7217-11e7-9899-67fa3d74ca8e.png)

**Comment by urikphytech** [comment]
Yes
I also tried adding `$(SRCROOT)/Pods (recursive)` to RNHeap's Header Search Paths

**Comment by negativetwelve** [comment]
hmm I was trying to reproduce this on a small react-native project but couldn't. I'll keep comparing the differences between my project and the readme, but if you could supply a reproducible project that would be super helpful

**Comment by negativetwelve** [comment]
Hey @urikphytech, have you tried this? https://github.com/facebook/react-native/issues/12042#issuecomment-274800537

Basically add React to the build phrase of your project, uncheck parallelize builds, then move React to build before your project. Let me know if that helps

**Comment by negativetwelve** [comment]
hey @urikphytech just checking in, was the above solution above to make it work?

**Comment by urikphytech** [comment]
Hey @negativetwelve, thanks for you help. Due to time constraints I ended up integrating mixpanel so I hadn't had the chance to try your suggestion. I promise to give it a try once I have some free time. 

Again, thanks a lot for your help!

**Comment by negativetwelve** [comment]
no worries, thanks for following up!

**Comment by negativetwelve** [comment]
Hi @urikphytech, just wanted to follow up on this because I reproduced it in another project I'm working on!

Turns out, if you have the `react-native-heap-analytics` package installed via `react-native link` and cocoapods, you'll end up with two copies. The `react-native link` version installed in the project file under `Libraries` won't be able to find the reference to `Heap`. If you just remove `RNHeap` from the `Libraries` directory in Xcode, that should fix the problem.

Let me know if you have a chance to try it out! I'm going to close this for now since that fixed the issue for me!
